doc/tutorials/views/01_views.py

```diff
@@ -4,7 +4,7 @@
 # --- Constants ---
 SPRITE_SCALING_PLAYER = 0.5
 SPRITE_SCALING_COIN = .25
-COIN_COUNT = 50
+COIN_COUNT = 3
```

Comment:
I kind of like keeping the 50 coins. Yes, this is quicker but 50 coins doesn't take long to gather. Otherwise you don't get much of a game at all. And if the three coins spawn on top of the player? You'll switch to end game before you know what is happening.

Reply:
Would 25 be ok as a middle ground, or is the goal to get users to adjust the value themselves?

Reply:
25 doesn't bother me.